Header header
geometry_msgs/Pose pose
string class_label
float32 confidence
float32 xmin
float32 ymin
float32 xmax
float32 ymax 
